bodhi/models/models.py

```diff
@@ -1649,19 +1649,35 @@ def check_requirements(self, session, settings):
 
         return True, "All checks pass."
 
+    def negative_karma(self):
+        """Checks for negative karma, Returns True if the update has negative karma"""
+        feedback = defaultdict(int)
```

Comment:
Also a nitpick that you can ignore, but consider using ```defaultdict(lambda: 0)``` here instead simply for being more explicit. You can leave it as is if you prefer.